### PR TITLE
Setting Codewind server logging levels

### PR DESCRIPTION
**Pull request by sishida**
PR that touches the following issues:

Developer issue: eclipse/codewind#1251
Doc issue that corresponds to the developer issue: eclipse/codewind#2227
Related doc issue: eclipse/codewind#2141

### CODE REVIEW

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -292,6 +292,17 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
...
+<!--
+Action/Topic: Understanding application metrics
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, change the default log level, such as trace, in PFE with the `cwctl loglevels` command.
+
+In Eclipse, set your preference to enable support features. If you enable the preference, the menus include setting PFE log level items.
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: In VS Code, use the `Codewind: Set Codewind Server Logging Level` command in the Command Palette.

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -292,6 +292,17 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
...
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
+<!--
+Action/Topic: Understanding application metrics
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, change the default log level, such as trace, in PFE with the `cwctl loglevels` command.
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: To assist with problem determination, raise the default Codewind server log level to Debug or Trace with the `cwctl loglevels` command.

File: docs/_documentations/troubleshooting.md (hunk 1 above)
Comment (eharris369) [reply docs/_documentations/troubleshooting.md]: In Eclipse, use the Codewind preference page to enable support features. Once enabled, you can right click on a connection in the Codewind Explorer view and select **Codewind server log level** to change the server logging level.

File: docs/_documentations/troubleshooting.md (hunk 2 above)
Comment (eharris369) [inline docs/_documentations/troubleshooting.md]: Should we refer to PFE or just call it the Codewind server?

Comment (tetchel) [reply docs/_documentations/troubleshooting.md]: I agree that we should not use the term 'PFE'. it refers to 'the codewind server' which could also be called 'the codewind container' but in our UI we call it the former so I think that's the best term for it.

**Comment by sishida** [comment]
@eharris369 @tetchel Thank you for your feedback. I have incorporated it into the latest commit. :)

File: docs/_documentations/troubleshooting.md [hunk 3]
```diff
@@ -292,6 +292,18 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
...
+<!--
+Action/Topic: Understanding application metrics
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace** with the `cwctl loglevels` command.
+
+- In Eclipse, set your preference to enable support features. If you enable the preference, the menus include setting the Codewind server log level items.
```
Comment (eharris369) [inline docs/_documentations/troubleshooting.md]: I still think we should tell the user to right click on the connection in the Codewind Explorer view so they know what menu to use or `the connection menu in the Codewind Explorer view`.  Just saying `the menus` is confusing.

File: docs/_documentations/troubleshooting.md [hunk 4]
```diff
@@ -292,6 +292,18 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
 
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
+<!--
+Action/Topic: Understanding application metrics
```
Comment (eharris369) [inline docs/_documentations/troubleshooting.md]: John mentioned that this does not look like the right Action/Topic for the server log level setting.

File: docs/_documentations/troubleshooting.md [hunk 5]
```diff
@@ -299,9 +299,10 @@ Issue link: https://github.com/eclipse/codewind/issues/1251
 Info added in 0.10.0.
 -->
 ## Setting Codewind server logging levels
-To assist with problem determination, change the default log level, such as trace, in PFE with the `cwctl loglevels` command.
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace** with the `cwctl loglevels` command.
```
Comment (tetchel) [inline docs/_documentations/troubleshooting.md]: we should make it clear that the `cwctl loglevels` does the exact same thing as the IDE instructions below

File: docs/_documentations/troubleshooting.md (hunk 4 above)
Comment (sishida) [reply docs/_documentations/troubleshooting.md]: @eharris369 Thanks for reminding me! What about putting it under "Checking the application and build statuses"?

**Comment by sishida** [comment]
Latest feedback merged.

File: docs/_documentations/troubleshooting.md [hunk 6]
```diff
@@ -309,6 +309,18 @@ If you modify files in MicroProfile projects, sometimes the project gets double
...
+
+<!--
+Action/Topic: Checking the application and build statuses
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace**. Use the `cwctl loglevels` command or follow the instructions for an IDE:
+- In Eclipse, set your preference to enable support features. Right-click the connection or the connection menu in the Codewind Explorer view. Now, the menus include setting the Codewind server log level items.
```
Comment (eharris369) [inline docs/_documentations/troubleshooting.md]: How about:
In Eclipse, enable support features in the Codewind preferences, then right-click the connection in the Codewind Explorer view and select **Codewind server log level**.
I think we need to be clear about which preferences as Eclipse has a lot of preferences.

File: docs/_documentations/troubleshooting.md (hunk 4 above)
Comment (eharris369) [reply docs/_documentations/troubleshooting.md]: Can it just go under its own topic?

Comment (sishida) [reply docs/_documentations/troubleshooting.md]: Yes, since that sounds like it is pretty unique, we can move it to its own section.

**Comment by sishida** [comment]
Add new section, edited Eclipse bullet point.

**Pull request merged**